subgraphs/gmx/src/entities/pool.ts

```diff
@@ -0,0 +1,546 @@
+      break;
+    case EventType.CollateralIn:
+      pool.cumulativeEntryPremiumUSD =
+        pool.cumulativeEntryPremiumUSD.plus(amountUSD);
+      pool.cumulativeTotalPremiumUSD =
+        pool.cumulativeTotalPremiumUSD.plus(amountUSD);
+      break;
+    case EventType.CollateralOut:
+      pool.cumulativeExitPremiumUSD =
+        pool.cumulativeExitPremiumUSD.plus(amountUSD);
+      pool.cumulativeTotalPremiumUSD =
+        pool.cumulativeTotalPremiumUSD.plus(amountUSD);
+      break;
+
+    default:
+      break;
+  }
+  pool._lastUpdateTimestamp = event.block.timestamp;
+  pool.save();
+
+  increaseProtocolPremium(event, amountUSD, eventType);
+}
+
+export function incrementPoolOpenPositionCount(
+  event: ethereum.Event,
+  pool: LiquidityPool,
+  positionSide: string
+): void {
+  if (PositionSide.LONG == positionSide) {
+    pool.longPositionCount += INT_ONE;
+  } else {
+    pool.shortPositionCount += INT_ONE;
+  }
+  pool.openPositionCount += INT_ONE;
+
+  pool._lastUpdateTimestamp = event.block.timestamp;
+  pool.save();
+
+  incrementProtocolOpenPositionCount(event, positionSide);
+}
+
+export function decrementPoolOpenPositionCount(
+  event: ethereum.Event,
+  pool: LiquidityPool,
+  positionSide: string
+): void {
+  if (PositionSide.LONG == positionSide) {
+    pool.longPositionCount =
+      pool.longPositionCount - INT_ONE >= INT_ZERO
+        ? pool.longPositionCount - INT_ONE
+        : INT_ZERO;
+  } else {
+    pool.shortPositionCount =
+      pool.shortPositionCount - INT_ONE >= INT_ZERO
+        ? pool.shortPositionCount - INT_ONE
+        : INT_ZERO;
+  }
+  pool.openPositionCount =
+    pool.openPositionCount - INT_ONE >= INT_ZERO
+      ? pool.openPositionCount - INT_ONE
+      : INT_ZERO;
+  pool.closedPositionCount += INT_ONE;
+
+  pool._lastUpdateTimestamp = event.block.timestamp;
+  pool.save();
+
+  decrementProtocolOpenPositionCount(event, positionSide);
+}
+
+export function updatePoolOutputToken(
+  event: ethereum.Event,
+  pool: LiquidityPool,
+  outputTokenAddress: Address
+): void {
+  pool.outputToken = getOrCreateToken(event, outputTokenAddress).id;
+  pool._lastUpdateTimestamp = event.block.timestamp;
+  pool.save();
+}
+
+export function updatePoolTvl(
+  event: ethereum.Event,
+  pool: LiquidityPool,
+  aumUSD: BigDecimal,
+  outputTokenSupply: BigInt
+): void {
+  const prevPoolTVL = pool.totalValueLockedUSD;
+  pool.totalValueLockedUSD = aumUSD;
+  const tvlChangeUSD = pool.totalValueLockedUSD.minus(prevPoolTVL);
+
+  pool.outputTokenSupply = outputTokenSupply;
+
+  const outputToken = getOrCreateToken(
+    event,
+    Address.fromBytes(pool.outputToken!)
+  );
+  if (outputTokenSupply == BIGINT_ZERO) {
+    pool.outputTokenPriceUSD = BIGDECIMAL_ZERO;
+  } else {
+    pool.outputTokenPriceUSD = pool.totalValueLockedUSD.div(
+      convertTokenToDecimal(outputTokenSupply, outputToken.decimals)
+    );
+  }
+  pool.stakedOutputTokenAmount = pool.outputTokenSupply;
```

Comment:
It is. The protocol stake GLP token for users automatically as shown at https://github.com/gmx-io/gmx-contracts/blob/65e62b62aadea5baca48b8157acb9351249dbaf1/contracts/staking/RewardRouter.sol#L117-L128

It can also be confirmed from GMX website at https://app.gmx.io/#/dashboard, as shown below:
<img width="394" alt="image" src="https://user-images.githubusercontent.com/28422812/218980583-67fd7421-e453-486c-887c-27554b643a86.png">